controllers/signout.js

```diff
@@ -0,0 +1,14 @@
+const signout = (req, res) => {
+  const cookies = Object.keys(req.cookies);
+  cookies.forEach((cookie) => {
+    res.clearCookie(cookie);
+  });
```

Comment:
Why are we deleting all cookies?
Let's say we have 2 cookies. First cookie is for auth and other for some other purpose other than auth, say to store user's theme preference (dark/light)
Now if the user signs out you'll loose the 2nd cookie also which should not have happened